workspaces/web-app/pages/_document.js

```diff
@@ -1,11 +1,9 @@
 import Document, { Head, Main, NextScript } from 'next/document'
```

Comment:
Do we then still want to keep `_document.js`? Maybe we should remove it?

Reply:
Removed it